main.js

```diff
@@ -124,6 +130,57 @@ for (var i = 0 ; i < courses.length ; i++) {
    }
 }
 
+sortNameDown.style.display="none";
+withoutSort.style.display="none";
+
+sortName.onclick = function(){
+    sortElement = courses.slice();
+   courses.sort(function (a, b) {
+        if (a.name < b.name) {
+          return -1;
+        }
+        if (a.name > b.name) {
+          return 1;
+        }
+        return 0;
+      });
+      
+      console.log(courses); 
+      display();
+      sortName.style.display="none";
+      sortNameDown.style.display="inline-block";
```

Comment:
I see sortName.onclick and sortNameDown.onclick, are almost the same functionality but with an oposite behaviour, can you refactor the code to have only one function and reuse it based if you want to sort up or down

Reply:
the code update by one function